src/pages/projectSettings/tabs/PluginsTab/getFormSchema.tsx

```diff
@@ -128,6 +129,28 @@ export const getFormSchema = (
                       },
                     },
                   },
+                  ticketCreateIssueType: {
+                    type: "object" as "object",
+                    title: "Ticket Create Issue Type",
+                    properties: {
+                      issueType: {
+                        type: "string" as "string",
+                        title: "",
+                        oneOf: [
+                          JiraTicketType.Investigation,
+                          JiraTicketType.Bug,
+                          JiraTicketType.Task,
+                          JiraTicketType.BuildFailure,
+                          JiraTicketType.Epic,
+                          JiraTicketType.Incident,
+                        ].map((r: string) => ({
```

Comment:
how about this, to avoid potential manual updates in the future:

```suggestion
                        oneOf: Object.values(JiraTicketType).map((r: string) => ({
```